sfdc37/salesforce_connector.bal

```diff
@@ -18,272 +18,266 @@
+    # + return - Opportunity ID if successful else SalesforceConnectorError occured
     public function createOpportunity(json opportunityRecord) returns (string|SalesforceConnectorError);
 
-    documentation {Deletes existing Opportunity's records
-        P{{opportunityId}} Opportunity ID
-        R{{}} true if successful else false or SalesforceConnectorError occured.}
+    # Deletes existing Opportunity's records.
+    # + opportunityId - Opportunity ID
+    # + return - true if successful else false or SalesforceConnectorError occured
     public function deleteOpportunity(string opportunityId) returns (boolean|SalesforceConnectorError);
 
-    documentation {Updates existing Opportunity object record
-        P{{opportunityId}} Opportunity ID
-        R{{}} true if successful else false or SalesforceConnectorError occured.}
+    # Updates existing Opportunity object record.
+    # + opportunityId - Opportunity ID
+    # + return - true if successful else false or SalesforceConnectorError occured
     public function updateOpportunity(string opportunityId, json opportunityRecord)
                         returns (boolean|SalesforceConnectorError);
 
     //Product
-    documentation {Accesses Products SObject records based on the Product object ID
-        P{{productId}} Product ID
-        R{{}} Json result if successful else SalesforceConnectorError occured.}
+    # Accesses Products SObject records based on the Product object ID.
+    # + productId - Product ID
+    # + return - Json result if successful else SalesforceConnectorError occured
     public function getProductById(string productId) returns (json|SalesforceConnectorError);
 
-    documentation {Creates new Product object record
-        P{{productRecord}} JSON product record
-        R{{}} Product ID if successful else SalesforceConnectorError occured.}
+    # Creates new Product object record.
+    # + productRecord - JSON product record
+    # + return - Product ID if successful else SalesforceConnectorError occured
     public function createProduct(json productRecord) returns (string|SalesforceConnectorError);
 
-    documentation {Deletes existing product's records
-        P{{productId}} Product ID
-        R{{}} true if successful else false or SalesforceConnectorError occured.}
+    # Deletes existing product's records.
+    # + productId - Product ID
+    # + return - true if successful else false or SalesforceConnectorError occured
     public function deleteProduct(string productId) returns (boolean|SalesforceConnectorError);
 
-    documentation {Updates existing Product object record
-        P{{productId}} Product ID
-        P{{productRecord}} JSON product record
-        R{{}} Json result if successful else SalesforceConnectorError occured.}
+    # Updates existing Product object record.
+    # + productId - Product ID
+    # + productRecord - JSON product record
+    # + return - Json result if successful else SalesforceConnectorError occured
```

Comment:
Json -> JSON
Shall we correct in all the places? If referring to the data type in Ballerina, then use `json`